<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Joachim Ansorg</author>
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>1154</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-390</y>
         <width>329</width>
         <height>984</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="titleEdit">
          <property name="placeholderText">
           <string>Title of the report</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="descriptionEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Your description to display at the top of the report. HTML content is allowed.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Project</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="ProjectTreeComboBox" name="projectsBox"/>
           </item>
           <item>
            <widget class="QCheckBox" name="subprojectsCheckbox">
             <property name="text">
              <string>Include subprojects</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="entriesGroupBox">
          <property name="title">
           <string>Time Entries</string>
          </property>
          <layout class="QVBoxLayout" name="entriesBoxLayout">
           <item>
            <widget class="QCheckBox" name="roundEntriesCheckBox">
             <property name="text">
              <string>Round entries</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="frameRoundingMinutes">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="frameRoundingValueGroup" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QComboBox" name="frameRoundingMode"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>to</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="frameRoundingValue">
                  <property name="suffix">
                   <string/>
                  </property>
                  <property name="maximum">
                   <number>180</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="frameRoundingUnit"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="dateFilterCheckbox">
             <property name="text">
              <string>Filter by date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="dateFilterGroup" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QDateEdit" name="dateStart">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>to</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEnd">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="includeArchivedCheckBox">
             <property name="text">
              <string>Include archived entries</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="splitGroupBox">
          <property name="title">
           <string>Splitting</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="splitLayout">
             <item>
              <widget class="QListView" name="splitList">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>160</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::NoDragDrop</enum>
               </property>
               <property name="defaultDropAction">
                <enum>Qt::CopyAction</enum>
               </property>
               <property name="flow">
                <enum>QListView::TopToBottom</enum>
               </property>
               <property name="resizeMode">
                <enum>QListView::Adjust</enum>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="splitMoveUp">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset theme="go-up">
                   <normaloff>../../../../../../../home/jansorg/.designer/backup</normaloff>../../../../../../../home/jansorg/.designer/backup</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string>Alt+Up</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="splitMoveDown">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset theme="go-down">
                   <normaloff>../../../../../../../home/jansorg/.designer/backup</normaloff>../../../../../../../home/jansorg/.designer/backup</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string>Alt+Down</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>10</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="reportGroupBox">
          <property name="title">
           <string>&amp;Template</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QComboBox" name="templateBox"/>
           </item>
           <item>
            <widget class="QCheckBox" name="showIsoDatesCheckbox">
             <property name="text">
              <string>Show ISO dates</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showEmptyCheckbox">
             <property name="text">
              <string>Show empty items in groups</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showSummaryCheckbox">
             <property name="text">
              <string>Show summary</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="matrixTablesCheckbox">
             <property name="text">
              <string>Matrix tables</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showSalesCheckbox">
             <property name="text">
              <string>Show sales</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showTrackedCheckbox">
             <property name="text">
              <string>Show tracked averages</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showUntrackedCheckbox">
             <property name="text">
              <string>Show untracked averages</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="useDecimalTimeFormat">
             <property name="text">
              <string>Use decimal time format</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showStopTimeCheckbox">
             <property name="text">
              <string>Show end time (timelog reports)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="cssFileLayout">
             <item>
              <widget class="QLabel" name="cssFileLabel">
               <property name="toolTip">
                <string>Optional path to a file with CSS. The file will be inlined into the generated report.</string>
               </property>
               <property name="text">
                <string>CSS file:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cssFileEdit">
               <property name="placeholderText">
                <string>Optional path to a CSS file</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="previewFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>10</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="previewFrameLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QToolButton" name="actionsButton">
       <property name="text">
        <string>&amp;Update Report</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTreeComboBox</class>
   <extends>QComboBox</extends>
   <header>commonWidgets/ProjectTreeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateFilterCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateFilterGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roundEntriesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameRoundingValueGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateReport()</slot>
 </slots>
</ui>
